tstest/integration/vms/harness_test.go

```diff
@@ -96,7 +96,12 @@ func newHarness(t *testing.T) *Harness {
 	hs := &http.Server{Handler: mux}
 	go hs.Serve(ln)
 
-	run(t, dir, "ssh-keygen", "-t", "ed25519", "-f", "machinekey", "-N", ``)
+	cmd := exec.Command("ssh-keygen", "-t", "ed25519", "-f", "machinekey", "-N", ``)
```

Comment:
I prefer to only use backticks when in regexps or when they're really needed. I'd prefer `""` there, but minor.